osu.Game/Overlays/News/NewsHeader.cs

```diff
@@ -4,20 +4,24 @@
 using System;
 using osu.Framework.Bindables;
 using osu.Framework.Graphics;
+using osu.Framework.Localisation;
+using osu.Game.Localisation;
 
 namespace osu.Game.Overlays.News
 {
     public class NewsHeader : BreadcrumbControlOverlayHeader
     {
-        private const string front_page_string = "frontpage";
+        public static LocalisableString FrontPageString => NewsStrings.FrontPageString;
+        public static LocalisableString HeaderTitle => NewsStrings.HeaderTitle;
+        public static LocalisableString HeaderDescription => NewsStrings.HeaderDescription;
```

Comment:
Should be `public static readonly`.

Reply:
`readonly` is invalid for this.
this code format is the same as other localisation PR.

Reply:
Fair, misread.